proto/dydxprotocol/clob/order.proto

```
@@ -1,6 +1,7 @@
 syntax = "proto3";
 package dydxprotocol.clob;
 
+import "cosmos_proto/cosmos.proto";
```

Buf (1.54.0)

import "cosmos_proto/cosmos.proto": file does not exist

(COMPILE)